ci_framework/roles/libvirt_manager/tasks/attach_interface.yml

```diff
@@ -14,21 +14,39 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-
 # This task requires
 #   vm_name         Domain name to which the interface needs to be attached.
-#   interface_name  source of the device
-#   interface_type  bridge or network
+#   network
+#     name          Name of the network bridge
+#     cidr          network notation
+#     mtu           MTU applied on the interface.
+#     static_ip     boolean if static address is required.
+
+
+- name: Generate a random MAC address
+  ansible.builtin.set_fact:
+    _lm_mac_address: "{{ '0A:02' | community.general.random_mac }}"
```

Comment:
is OA:02 a special OUI or did you just "choose" something. 
I would have expected it to be 3 octets though (eg OA:02:01)

Reply:
`xA` could enable us identify that it is a private MAC address range. Did

https://www.oasys.net/fragments/identify-private-macs/